README.md

```diff
@@ -140,7 +140,7 @@ Therefore, the app does not collect any data without your consent. NewPipe's pri
 ## License
 [![GNU GPLv3 Image](https://www.gnu.org/graphics/gplv3-127x51.png)](https://www.gnu.org/licenses/gpl-3.0.en.html)  
 
-NewPipe is Free Software: You can use, study share and improve it at your
+NewPipe is Free Software: You can use, study, share, and improve it at your
```

Comment:
```suggestion
NewPipe is Free Software: You can use, study, share, and improve it at
```

Reply:
Eh, I like the original better.

Reply:
Lol, it's a correction, not an alternative. The original is wrong.

Reply:
It looks correct to me. Mind explaining the mistake?

Reply:
The phrase is "at will": https://en.wiktionary.org/wiki/at_will

I can't seem to find anything relevant for "at your will" because the dumb search engine thinks I'm asking about legal wills.

Here is an informal source, though: https://forum.wordreference.com/threads/at-your-will.2687970/

Reply:
I'm aware of "at will," but I've never heard of "at your will" being incorrect, and the last comment in the thread you mentioned (which I happened to find myself by the way) says it's fine.

Reply:
The reason it jumped out at me as being odd is because I've never encountered "at your will" anywhere. The phrase "at will" is well known and used all the time, meanwhile.

Moreover, the preposition and pronoun combo seems semantically wrong. "As per your will" should be used instead, in that case.

Reply:
I'm not convinced, but this isn't important anyway. I'll change it.